src/Jackett.Common/Indexers/BrokenStones.cs

```diff
@@ -13,7 +13,7 @@ public class BrokenStones : GazelleTracker
         public override string Id => "brokenstones";
         public override string Name => "BrokenStones";
         public override string Description => "Broken Stones is a Private site for MacOS and iOS APPS / GAMES";
-        public override string SiteLink { get; protected set; } = "https://broken-stones.club/";
+        public override string SiteLink { get; protected set; } = "https://brokenstones.is/";
         public override string[] LegacySiteLinks => new[]
         {
             "https://brokenstones.club/"
```

Comment:
Please remember to add the old sitelink to the legacySiteLinks so that the indexer config can automatically update to the new domain
```suggestion
            "https://brokenstones.club/",
            "https://broken-stones.club/"
```
Thanks.